frontend/src/views/MovieDetailsScreen.vue

```diff
@@ -42,7 +42,7 @@ export default {
 
     async fetchMovie (id) {
       try {
-        const { data: movie } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movies-${id}`, {
+        const { data: movie } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movies-${id}/`, {
```

Comment:
this URL won't work wince movie-detail view endpoint is api/cinema/movies/${id}/ but you have api/cinema/movies-${id}/. Please correct